packages/sanity/src/structure/panes/document/documentPanel/header/perspective/DocumentPerspectiveMenu.tsx

```diff
@@ -21,33 +18,14 @@ const BadgeButton = styled(Button)({
 
 export function DocumentPerspectiveMenu(props: {documentId: string}): JSX.Element {
   const {documentId} = props
-  const client = useClient(DEFAULT_STUDIO_CLIENT_OPTIONS)
-  const {data} = useBundles()
   const {currentGlobalBundle} = usePerspective()
-  const bundles = useMemo(() => data ?? [], [data])
 
   const existsInBundle = getBundleSlug(documentId) === currentGlobalBundle?.slug
   const {title, hue, icon, slug} = currentGlobalBundle
+  const {data: documentVersions} = useDocumentVersions({documentId})
```

Comment:
Good point. I'll leave it to @RitaDias to decide whether to do that in a follow-up PR or before merging in this one.

Reply:
re `BundleActions` I think they will eventually change, specially after the demo, probably we will want it in a more obvious place, but that won't be decided until Marius is back.
Not sure how that change will impact this decision, but I think moving it to the PaneProvider will make it available for all the possible consumers. It can be done in a follow-up PR once you are back from support

Reply:
WIll create a Linear story since I'll be on holiday next week 👍

Reply:
Linear story created 👍